source/change-streams/tests/change-streams.yml

```diff
@@ -0,0 +1,391 @@
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            y: 2
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            z: 3
+    expectations:
+      -
+        command_started_event:
+
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              - 
+                $changeStream:
+                  fullDocument: default
+              -
+                $match:
+                  "fullDocument.z":
+                    $numberInt: "3"
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success:
+        -
+          operationType: insert
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            z:
+              $numberInt: "3"
+  -
+    description: Change Stream should error when an invalid aggregation stage is passed in
+    minServerVersion: "3.6.0"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline:
+      -
+        $darmokAndJaladAtTanagra: SHAKA WHEN THEN WALLS FELL
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            z: 3
+    expectations:
+      -
+        command_started_event:
+
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              - 
+                $changeStream:
+                  fullDocument: default
+              -
+                $darmokAndJaladAtTanagra: SHAKA WHEN THEN WALLS FELL
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      error:
+        errmsg: "Unrecognized pipeline stage name: '$darmokAndJaladAtTanagra'"
+        code: 40324
+
+  -
+    description:  A fresh ChangeStream against a server >=4.0 will always include `startAtOperationTime` in the `$changeStream` stage.
+    minServerVersion: "4.0.0-rc0"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline: []
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            x: 1
+    expectations:
+      - 
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              - 
+                $changeStream:
+                  fullDocument: default
+                  startAtOperationTime:
+                    $timestamp:
+                      i: 42
+                      t: 42
+          command_name: aggregate
+          database_name: *database_name
+    result: {}
```

Comment:
Another instance of not asserting a result. If this is a valid test case where an APM check is needed then the README should be updated to discuss this.